crates/biome_js_analyze/src/semantic_analyzers/style/use_naming_convention.rs

```diff
@@ -338,6 +343,23 @@ impl Rule for UseNamingConvention {
         } else {
             markup! {""}.to_owned()
         };
+
+        if options.strict_case {
+            let case_type = Case::identify(name, false);
+            let case_strict = Case::identify(name, true);
+            if case_type == Case::Camel && case_strict == Case::Unknown {
+                return Some(RuleDiagnostic::new(
+                    rule_category!(),
+                    ctx.query().syntax().text_trimmed_range(),
+                    markup! {
+                        "If `strict` is `true`, then two consecutive uppercase characters are not allowed in camelCase and PascalCase."
```

Comment:
The option is named `strictCase`. Also, since the option is turned on, we should not use `if`?
For example:

```suggestion
                        "Two consecutive uppercase characters are not allowed in camelCase and PascalCase because `strictCase` is set to `true`."
```